ember.md

```diff
@@ -47,9 +47,12 @@ lastName: attr('string'),
 fullName: belongsTo('person')
 ```
 
-- 1.3 Default Unassigned Property
-When declaring and unassigned property we should default it to `null` instead of `undefined`
+- 1.3 **Default Unassigned Property**
 
+  + When declaring an unassigned property we should default it to `null` instead of `undefined`
+
+  + Why? Because defining a property as `undefined` has the meaning of we didn't define it. By setting it to `null` you are explicitly telling the app that the property has been declared
```

Comment:
Maybe it needs to be expanded in the documentation since it was unclear to @sernaashui , but the idea is that if we are explicitly declaring vars then we should not be initializing them to the value that means 'this variable has not been declared'.

Reply:
Ok. That makes sense. Maybe I misunderstood how this standard came to be. I wasn't aware of any vars being initialized as `undefined`. I thought setting the initial value to `null` was to replace initial values set as an empty string.

Reply:
Yeah I think the original item was for vars we are declaring in our components, but they don't have an actual default value (like `''` or `0`). In that case we can use `null` to indicate that the var has been defined, but it hasn't yet been used. For components I think we have to give a value. In code we see this:

```javascript
var test
typeof test
// "undefined"
typeof notDefined
// "undefined"
```

To my thinking it's unhelpful to define `test` without somehow differentiating it from a truly undefined variable.

Reply:
I agree. Thanks for the clarification.